quarkus-graalvm-builder-image/graalvm.yaml

```diff
@@ -1,57 +1,18 @@
 images:
-  - graalvm-version: 22.2.0
-    java-version: 11
-    tag: 22.2-java11
+  # https://github.com/graalvm/graalvm-ce-builds/releases/tag/jdk-17.0.7
+  - java-version: 17.0.7
+    tags: 17, 17.0
```

Comment:
Quoting from our group chat

> ...agreed on using tags of the form: 23.0.1-jdk-17, jdk-17 and jdk-17.0.7...

Does 17.0 apply?

Reply:
I needed multiple tags (to test the support). 
However, you are right about the "jdk-" part. It should be jdk-17. 

The current result is:
https://quay.io/repository/cescoffi/ubi-quarkus-graalvmce-builder-image?tab=tags